rpc.go

```diff
@@ -113,11 +116,24 @@ func formatRaw(msg string, args []*v1.LogAttrs) string {
 	return fmt.Sprintf("%s %s", msg, res[:len(res)-2]) // remove last comma
 }
 
-func format(args []*v1.LogAttrs) []zap.Field {
+func (r *RPC) format(args []*v1.LogAttrs) []zap.Field {
```

Comment:
No need to be on the RPC structure, you may use `format(cfg.option, args ...*v1.LogAttrs)` to avoid double packing here.

Reply:
fixed